using_images/s2i_images/customization.adoc

```diff
@@ -0,0 +1,84 @@
+[[using-s2i-images]]
+= Using S2I images
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+S2I builder images normally include
+xref:../creating_images/s2i.html#s2i-scripts[*_assemble_* and *_run_* scripts],
+but the default behavior of those scripts may not be suitable for all users.
+This document will cover a few approaches for customizing the behavior of an
+S2I builder that includes default scripts.
+
+== Invoking scripts embedded in an image
+
+Typically builder images provide their own version of the S2I scripts that
+cover the most common use-cases. If these scripts don't fulfill your needs, S2I
+provides a way of overriding them by adding custom ones in the *_.s2i/bin_*
+directory. However, by doing this you are completely replacing the standard
```

Comment:
we have docs about how to override the image scripts (by putting them in your repo or by referencing an external url), we should link to those docs here.